lib/main.dart

```diff
@@ -1,10 +1,8 @@
 import 'package:cloud_firestore/cloud_firestore.dart';
-import 'package:firebase_installations/firebase_installations.dart';
 import 'package:flutter/material.dart';
 import 'package:firebase_core/firebase_core.dart';
 import 'package:firebase_auth/firebase_auth.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
-import 'package:firebase_in_app_messaging/firebase_in_app_messaging.dart';
```

Comment:
I don't think notifications are going to show up if you remove these. It was in the instructions to add the import for in-app messaging.